docs/source/reporters.rst

```diff
@@ -231,33 +231,37 @@ You can have notifications sent to you through the `Matrix protocol`_.
 To achieve this, you first need to register a Matrix account for the bot
 on any homeserver.
 
-You then need to acquire an access token and room ID, using the
-following instructions adapted from `this
-guide <https://t2bot.io/docs/access_tokens/>`__:
-
-1. Open `Riot.im <https://riot.im/app/>`__ in a private browsing window
-2. Register/Log in as your bot, using its user ID and password.
-3. Set the display name and avatar, if desired.
-4. In the settings page, select the "Help & About" tab, scroll down to the bottom and click Access
-   Token: <click to reveal>.
-5. Copy the highlighted text to your configuration.
-6. Join the room that you wish to send notifications to.
-7. Go to the Room Settings (gear icon) and copy the *Internal Room ID*
-   from the bottom.
-8. Close the private browsing window **but do not log out, as this
-   invalidates the Access Token**.
+You then need to acquire room ID, using the following instructions:
+
+#. Open `Riot.im <https://riot.im/app/>`__ in a private browsing window
+#. Register/Log in as your bot, using its user ID and password.
+#. Set the display name and avatar, if desired.
+#. Join the room that you wish to send notifications to.
+#. Go to the Room Settings (gear icon) and copy the *Internal Room ID* from the bottom.
 
 Here is a sample configuration:
 
 .. code:: yaml
 
    matrix:
      homeserver: https://matrix.org
-     access_token: "YOUR_TOKEN_HERE"
      room_id: "!roomroomroom:matrix.org"
+     user: @botusername:matrix.org
      enabled: true
 
-You will probably want to use the following configuration for the
+Then you need to manually trigger connection to the matrix from urlparse
+to obtain ``access_token`` and generate encryption keys:
+
+.. code:: bash
+
+    $ urlwatch --test-reporter matrix
+
+This is one time process during which you will be prompted for a password
+to the bot account (password will not be stored).
+Obtained credentials and encryption keys will be stored in user data directory
+approprite for your system (e.g. ``~/.local/share/urlwatch/`` on linux).
```

Comment:
"approprite" -> "appropriate"

Reply:
Remove "(e.g. ``~/.local/share/urlwatch/`` on linux)" and replace with something like "(relative to urlwatch's configuration directory)".